tests/unit/models/primo.js

```diff
@@ -617,7 +618,120 @@ describe("Primo Model >", function() {
     });
   });
 
-  describe("primo model buildTemplate method >", function() {
+  describe("primo model isDefaultCoverImage method >", function() {
+    it("should return false when an actual coverImageUrl is returned by the API", function() {
+      var coverImageUrl = "https://assets.thirdiron.com/images/covers/0028-4793.png";
+      expect(primo.isDefaultCoverImage(coverImageUrl)).toEqual(false);
+    });
+
+    it("should return true when a default coverImageUrl is returned by the API", function() {
+      var coverImageUrl = "https://assets.thirdiron.com/images/covers/default.png";
+      expect(primo.isDefaultCoverImage(coverImageUrl)).toEqual(true);
+    });
+  });
+
+  describe("primo model getDirectToPDFUrl method >", function() {
+    it("should not include a fullTextFile property in the BrowZine API response for a journal", function() {
```

Comment:
I'd suggest a little rearrangement of the test here to match the given / when / then sort of format that is [popular with Cucumber](https://docs.cucumber.io/gherkin/reference/#steps) and generally [a BDD-style approach](https://martinfowler.com/bliki/GivenWhenThen.html).

How about something sort of like this?
```
describe("primo model getDirectToPDFUrl method >", function() {
  describe("given an API response for a journal", ...
    var scope = { ... issn: ["0096-6762" ...
    it("should not include a fullTextFile property", ...
```

I was at first unclear about where the test setup was and what the fixture was used for.  

After reading the test again, I can see the fixture is all about a journal response - and yup, journal responses don't have `fullTextFile` properties! 😊 

Typically we have fixtures setup in a `describe` setup, since the `describe` is usually a `given` or `when`.

What do you think of this approach, @tiandavis ?  Also, we could address this after this PR is merged if we'd like to get this deployed for John's demo tomorrow.

Thanks!

Reply:
@karlbecker Fantastic question @karlbecker! TLDR; is this is how we'd want to think about Acceptance/Integration Tests, not necessarily Unit Tests. E.g. notice our [Acceptance Tests](https://github.com/thirdiron/browzine-discovery-service-adapters/blob/develop/tests/acceptance/browzine-primo-adapter-test.js#L6) have that high-level journal vs article organization. Whereas, our Unit Test here is more granularly specific to a methods and sometimes those methods test for both journal or article use. 

I could see updating the `it` statement to focus more on what the method should be doing and not so much on the API details though. Will update the same.

Good point on Cucumber though. Noting, I wouldn't knock a development team that used Cucumber for their Acceptance Tests since there is the aspect of bridging the gap between development and business requirements. But, I my mind, the biggest benefit is using it at the higher Integration Testing Environment level since these business requirements typically involve multiple applications integrating to succeed on that higher level business requirement: e.g. apps here, `browzine-discovery-service-adapters` and `browzine_cms`.